src/Web/Shared/AssetSerials/GetAssetTypeSerialDto.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MRA.AssetsManagement.Web.Shared.AssetSerials
+{
+    public class GetAssetTypeSerialDto
```

Comment:
I think we don't use the *Dto suffix on naming DTO classes
Pls remove this suffix